internal/manifests/collector/adapters/config_from.go

```diff
@@ -5,16 +5,14 @@
 package adapters
 
 import (
-	"encoding/json"
 	"errors"
 
 	"gopkg.in/yaml.v2"
 )
 
 var (
 	// ErrInvalidYAML represents an error in the format of the configuration file.
-	ErrInvalidYAML = errors.New("couldn't parse the yaml configuration")
-	ErrInvalidJSON = errors.New("couldn't parse cloudwatch agent json configuration")
+	ErrInvalidYAML = errors.New("couldn't parse the opentelemetry-collector configuration")
```

Comment:
Should this be the cloudwatch agent config ?

Reply:
I think I missed the JSON validation logic here, good catch